haskell/src/Ligo/BaseDAO/Types.hs

```diff
@@ -870,12 +879,11 @@ mkFullStorage
   -> "metadata" :! TZIP16.MetadataMap
   -> "level" :! Natural
   -> "tokenAddress" :! Address
-  -> FullStorageSkeleton (VariantToExtra cep)
-mkFullStorage admin vp qt mcp cp gts extra mdt lvl tokenAddress = FullStorageSkeleton
-  { fsStorage = mkStorage admin extra mdt lvl tokenAddress (#quorumThreshold (argDef #quorumThreshold quorumThresholdDef qt))
-  , fsConfig  = mkConfig
+  -> StorageSkeleton (VariantToExtra cep)
+mkFullStorage admin vp qt mcp cp gts extra mdt lvl tokenAddress = let
```

Comment:
Probably, need renaming this function as well.

Reply:
Renamed